aip/0152.md

```diff
@@ -65,20 +66,34 @@ rpc RunWritingJob(RunWritingJobRequest)
   message. The errors themselves **must** still be represented with a
   [`google.rpc.Status`][] object.
 
-### Results
+### Executions and results
 
 Ordinarily, the API **should** provide results to the user as the final
 response of the `Run` method. However, this is sometimes insufficient; for
-example, a job that runs on a cadence in the background can not deliver results
-to the user in this way.
+example, a job that runs on a recurring schedule in the background can not
+deliver results to the user in this way.
 
-The service **may** store results as a sub-collection of resources under the
-job, which allows the user to list results of past job executions. A service
-that does this **should** define the `Get`, `List`, and `Delete` methods for
-the result resources.
+The service **may** store resources representing individual executions along
+with their result as a sub-collection of resources under the job, which allows
+the user to list past job executions. A service that does this **should**
+define the `Get`, `List`, and `Delete` methods for the execution resources:
+
+```proto
+message WriteBookJobExecution {
```

Comment:
Are we sold on executions? What about "runs" (some other GCP services do this IIRC)? Or something else? I'm not necessarily against "executions" but want to make sure we choose the right name since we'll have to live with it for a long time.

Reply:
I really dislike "run" as a noun personally, but I could be talked into it.

Reply:
I looked at our corpus and we pretty consistently use "run" as a verb, and we basically never use it as a noun, but we do not have a term that we _do_ use instead. (For the most part, we do not actually have "execution resources" like what is described here.)

Reply:
I can't think of anything besides run... So I guess we'll go with executions. :-)